README.md

```diff
@@ -1,27 +1,16 @@
 <div style="text-align:center">
   <img
-  src="https://raw.githubusercontent.com/ElrondNetwork/elrond-go/master/elrond_logo_01.svg"
-  alt="Elrond Network">
+  src="https://raw.githubusercontent.com/multiversx/mx-chain-go/master/multiversx-logo.svg"
```

Comment:
[question]

Was this logo svg file also renamed?

Reply:
https://github.com/multiversx/mx-chain-go/blob/master/multiversx-logo.svg